bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/NativeInterface.cs

```diff
@@ -150,6 +150,71 @@ internal static unsafe class NativeInterface
 
         internal unsafe struct CharPtr { };
 
+        #region Collections
+
+        internal static unsafe class LinkedList
+        {
+            internal unsafe struct LinkedListType { };
+
+            internal class LinkedListHandle
+                : ElectionguardSafeHandle<LinkedListType>
```

Comment:
and then wrap it in a `SafeHandle` which manages the lifecycle of the object (and tells the garbage collector to back off, from time to time).